docs/tutorials/webhook-provider.md

```diff
@@ -0,0 +1,44 @@
+# Webhook provider
+
+The "Webhook" provider allows to integrate ExternalDNS with DNS providers via an HTTP interface.
+The Webhook provider implements the Provider interface. Instead of implementing code specific to a provider, it implements an HTTP client that sends request to an HTTP API.
```

Comment:
```suggestion
The Webhook provider implements the `Provider` interface. Instead of implementing code specific to a provider, it implements an HTTP client that sends requests to an HTTP API.
```